packages/shopify-app-remix/docs/generated/generated_docs_data.json

```diff
@@ -1249,15 +1249,15 @@
   },
   {
     "name": "App proxy",
-    "description": "The `authenticate.public.appProxy` function validates app proxy requests made by Shopify, and returns a context to enable querying Shopify APIs.",
+    "description": "[App proxies](/docs/apps/online-store/app-proxies) take requests to Shopify links, and redirect them to external links. The `authenticate.public.appProxy` function validates requests made to app proxies, and returns a context to enable querying Shopify APIs.",
```

Comment:
@paulomarg Would this build the link correctly?

Reply:
Looks sane to me, it should support any MD tag.